### PR TITLE
Add authors links to pkgdown config

### PR DESCRIPTION
**Pull request by maelle**
To make names on https://r.igraph.org/index.html clickable

Cc @krlmlr

### CODE REVIEW

**Comment by maelle** [comment]
It'd be nice for everyone / most people listed here to check the URL associated to them (it could also be no URL).

File: _pkgdown.yml [hunk 1]
```diff
@@ -237,3 +237,19 @@ reference:
...
+
+authors:
+  Gábor Csárdi:
+    href: "https://github.com/gaborcsardi"
+  Tamás Nepusz:
+    href: "https://github.com/ntamas"
+  Vincent Traag:
+    href: "https://www.traag.net/"
+  Szabolcs Horvát:
+    href: "http://szhorvat.net/pelican/"
```
Comment (szhorvat) [inline _pkgdown.yml:249]:
```suggestion
    href: "http://szhorvat.net/"
```

**Review by ntamas — APPROVED** [review]

**Comment by maelle** [comment]
Does this need more reviews? :slightly_smiling_face:

**Comment by ntamas** [comment]
I don't think so, I'm just waiting for @krlmlr to merge this :)

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**